index.html

```diff
@@ -12,64 +12,56 @@
-        fill="#505F79" />
-    <path
-        d="M52.6448 3.03455C49.1561 3.03455 46.2891 5.86213 46.2891 9.37937V19.3104C46.2891 19.6897 46.5999 20.0001 46.9799 20.0001H50.5723C50.9523 20.0001 51.2632 19.6897 51.2632 19.3104V9.65524C51.2632 8.89661 51.8849 8.27592 52.6448 8.27592C53.4048 8.27592 54.0265 8.89661 54.0265 9.65524V19.3104C54.0265 19.6897 54.3374 20.0001 54.7174 20.0001H58.3098C58.6898 20.0001 59.0006 19.6897 59.0006 19.3104V9.37937C59.0006 5.89662 56.1682 3.03455 52.6448 3.03455Z"
-        fill="#505F79" />
-    <path
-        d="M41.487 0C40.1054 0 39 1.10345 39 2.48276C39 3.86207 40.1054 4.96552 41.487 4.96552C42.8687 4.96552 43.9741 3.86207 43.9741 2.48276C43.9741 1.10345 42.8687 0 41.487 0Z"
-        fill="#505F79" />
-    <path
-        d="M43.2832 6.27588H39.6908C39.3109 6.27588 39 6.58622 39 6.96553V19.3104C39 19.6897 39.3109 20 39.6908 20H43.2832C43.6632 20 43.9741 19.6897 43.9741 19.3104V6.96553C43.9741 6.58622 43.6632 6.27588 43.2832 6.27588Z"
-        fill="#505F79" />
-    <path
-        d="M136.985 7.46114C136.985 6.32124 136.619 5.25043 135.921 4.31779C136.187 3.41969 136.253 2.27979 136.187 0.967185C136.153 0.414508 135.721 0 135.189 0C134.889 0 132.36 0.0345423 130.896 1.38169C129.632 1.1399 128.301 1.1399 127.004 1.38169C125.573 0.0345423 123.044 0 122.712 0C122.179 0 121.747 0.414508 121.713 0.967185C121.614 2.27979 121.713 3.41969 121.98 4.31779C121.281 5.28497 120.915 6.35579 120.915 7.46114C120.915 9.8791 122.712 12.0553 125.44 13.0915C125.34 13.2988 125.274 13.5406 125.207 13.7824C122.312 13.4715 120.948 10.7427 120.882 10.639C120.649 10.1209 120.05 9.91364 119.551 10.19C119.052 10.4318 118.852 11.0535 119.118 11.5717C119.185 11.7444 121.015 15.4404 125.041 15.8549V18.9637C125.041 19.5509 125.473 20 126.039 20H131.861C132.427 20 132.86 19.5509 132.86 18.9637V14.8532C132.86 14.2314 132.726 13.6442 132.494 13.1261C135.189 12.0553 136.985 9.91364 136.985 7.46114Z"
-        fill="#505F79" />
-    <path
-        d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10205 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253175 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z"
-        fill="#505F79" />
-</svg>
+
+    <div class="background">
```

Comment:
Please use a `<section>` tag here.
It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.